Client/src/Pages/Settings Pages/StudentSettings/ProfileSettings/StudentProfileSettings.js

```diff
@@ -7,8 +7,26 @@ import {
   Container,
   TextField,
   Grid,
+  Fab
 } from "@material-ui/core";
 
+const text = {
+  marginBottom: 65
+}
+
+const button = {
```

Comment:
You should use better naming conventions here for making it easier for future developers to understand what this button is used for. Maybe something like saveButtonStyle or saveButtonPosition.

Reply:
Okay!

Reply:
@ishchhabra I have made some changes for now, give me feedback.

Reply:
The frontend fetches the first name and other values from the backend and pre-populates it with the current first name, lastname, ...
So, checking for length == 0 is not a correct check. You should instead store the fetched values from the backend into a separate variable (or maybe array) and compare the value with that. Also, I was hoping a good idea would be to use low opacity (with disabled) instead of completely hiding the button, as completely hiding the button might make the user confused as to where to click to save changes (before he actually made any).

Reply:
@ishchhabra Fixed, let me know what you feel now...

Reply:
Fixed @ishchhabra